modules/nf-core/gatk4/markduplicates/main.nf

```diff
@@ -25,13 +25,12 @@ process GATK4_MARKDUPLICATES {
 
     script:
     def args = task.ext.args ?: ''
-    prefix = task.ext.prefix ?: "${meta.id}.bam"
-
-    // If the extension is CRAM, then change it to BAM
-    prefix_bam = prefix.tokenize('.')[-1] == 'cram' ? "${prefix.substring(0, prefix.lastIndexOf('.'))}.bam" : prefix
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
```suggestion
    prefix = task.ext.prefix ?: "${meta.id}_md"
```
Set a default prefix like this, please don't hardcode _md in a way that can't be overridden